ecl/network/v2/load_balancer_syslog_servers/requests.go

```diff
@@ -0,0 +1,234 @@
+// `asc' or `desc'. Marker and Limit are used for pagination.
+type ListOpts struct {
+	Description    string `q:"description"`
+	ID             string `q:"id"`
+	IPAddress      string `q:"ip_address"`
+	LoadBalancerID string `q:"load_balancer_id"`
+	LogFacility    string `q:"log_facility"`
+	LogLevel       string `q:"log_level"`
+	Name           string `q:"name"`
+	PortNumber     int    `q:"port_number"`
+	Status         string `q:"status"`
+	TransportType  string `q:"transport_type"`
+}
+
+// ToLoadBalancerSyslogServersListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToLoadBalancerSyslogServersListQuery() (string, error) {
+	q, err := eclcloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+// List returns a Pager which allows you to iterate over a collection of
+// Load Balancer Syslog Servers. It accepts a ListOpts struct, which allows you to filter and sort
+// the returned collection for greater efficiency.
+//
+// Default policy settings return only those Load Balancer Syslog Servers that are owned by the tenant
+// who submits the request, unless the request is submitted by a user with
+// administrative rights.
+func List(c *eclcloud.ServiceClient, opts ListOpts) pagination.Pager {
+	url := listURL(c)
+	query, err := opts.ToLoadBalancerSyslogServersListQuery()
+	if err != nil {
+		return pagination.Pager{Err: err}
+	}
+	url += query
+	return pagination.NewPager(c, url, func(r pagination.PageResult) pagination.Page {
+		return LoadBalancerSyslogServerPage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// Get retrieves a specific Load Balancer Syslog Server based on its unique ID.
+func Get(c *eclcloud.ServiceClient, id string) (r GetResult) {
+	_, r.Err = c.Get(getURL(c, id), &r.Body, nil)
+	return
+}
+
+// CreateOpts represents the attributes used when creating a new Load Balancer Syslog Server.
+type CreateOpts struct {
+
+	// should syslog record acl info
+	AclLogging string `json:"acl_logging,omitempty"`
+
+	// should syslog record appflow info
+	AppflowLogging string `json:"appflow_logging,omitempty"`
+
+	// date format utilized by syslog
+	DateFormat string `json:"date_format,omitempty"`
+
+	// Description is description
+	Description string `json:"description,omitempty"`
+
+	// Ip address of syslog server
+	IPAddress string `json:"ip_address" required:"true"`
+
+	// The ID of load_balancer this load_balancer_syslog_server belongs to.
+	LoadBalancerID string `json:"load_balancer_id" required:"true"`
+
+	// 	Log facility for syslog
+	LogFacility string `json:"log_facility,omitempty"`
+
+	// Log level for syslog
+	LogLevel string `json:"log_level,omitempty"`
+
+	// Name is a human-readable name of the Load Balancer Syslog Server.
+	Name string `json:"name" required:"true"`
+
+	// Port number of syslog server
+	PortNumber int `json:"port_number,omitempty"`
+
+	// priority (0-255)
+	Priority int `json:"priority,omitempty"`
```

Comment:
If user sets 0 in Priority field, it will be omitted. So this should be pointer.
```suggestion
	Priority *int `json:"priority,omitempty"`
```